adapters/au_vic.js

```diff
@@ -0,0 +1,147 @@
+    }
+  });
+}
+
+var parameters = {
+  'PM2.5': 'pm25',
+  'PM10': 'pm10',
+  'NO2': 'no2',
+  'SO2': 'so2',
+  'O3': 'o3',
+  'CO': 'co',
+  'BC': 'bc'
+};
+
+var units = {
+  '&micro;g/m&sup3;': 'µg/m³',
+  'ppm': 'ppm'
+};
+
+var formatData = function (data, formatDataCB) {
+  var sites = JSON.parse(data).records;
+
+  // request measurements from each site
+  var tasks = sites.map(function (site) {
+    return function (cb) {
+      limiter.submit(request, {
+        url: `https://gateway.api.epa.vic.gov.au/environmentMonitoring/v1/sites/${site.siteID}/parameters`,
+        headers: {
+          'X-API-Key': process.env.EPA_VIC_TOKEN
+        }
+      }, function (err, res, body) {
+        if (err || res.statusCode !== 200) {
+          console.error(err, res);
+          return cb({message: 'Failure to load data url.'});
+        }
+        var source = JSON.parse(body);
+
+        // base properties shared for all measurements at this site
+        var baseProperties = {
+          location: source.siteName,
+          city: 'Victoria',
+          country: 'AU',
+          sourceName: source.name,
+          sourceType: 'government',
+          attribution: [{
+            name: 'EPA Victoria State Government of Victoria',
+            url: 'https://www.epa.vic.gov.au/EPAAirWatch'
+          }],
+          coordinates: {
+            latitude: source.geometry.coordinates[0],
+            longitude: source.geometry.coordinates[1]
+          }
+        };
+
+        // list of all measurements at this site
+        var measurements = [];
+        if (source && source.parameters) {
+          measurements = source.parameters.map(function (parameter) {
+            if (parameter.name in parameters) {
+              var measurement = baseProperties;
+              measurement.parameter = parameters[parameter.name];
+
+              // from the range of time series readings, find the 1HR_AV one
+              var averageReadings = parameter.timeSeriesReadings.filter(function (timeSeriesReading) {
+                return timeSeriesReading.timeSeriesName === '1HR_AV';
+              });
+
+              if (averageReadings.length && averageReadings[0].readings.length) {
+                var reading = averageReadings[0].readings[0];
+                if (reading.unit in units) {
+                  measurement.unit = units[reading.unit];
+                  measurement.averagingPeriod = { value: 1, unit: 'hours' };
+                  measurement.value = Number(reading.averageValue);
+
+                  var date = moment.tz(reading.since, 'Australia/Melbourne');
+
+                  measurement.date = {
+                    utc: date.toDate(),
+                    local: date.format()
+                  };
+
+                  return measurement;
+                }
+              }
+            }
+          }).filter(function (measurement) {
+            return measurement !== null;
+          });
+        }
+
```

Comment:
Oh thanks for picking up on that. I believe the issue was I needed to clone the base object instead of just assigning it which ended up replacing it each time.